src/sagemaker/image_uri_config/tensorflow.json

```diff
@@ -2629,6 +2661,39 @@
                     "us-west-2": "763104351884"
                 },
                 "repository": "tensorflow-training"
+            },
+            "2.7.0": {
```

Comment:
DLC is yet to release these containers. Can we remove this and introduce this later when DLC officially makes them available?